pub-critic/src/app/assets/menu-icon.svg

```diff
@@ -0,0 +1,8 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g id="Menu alt 4">
+<g id="Icon">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M3.59998 8.40001C3.59998 7.73727 4.13723 7.20001 4.79998 7.20001H19.2C19.8627 7.20001 20.4 7.73727 20.4 8.40001C20.4 9.06275 19.8627 9.60001 19.2 9.60001H4.79998C4.13723 9.60001 3.59998 9.06275 3.59998 8.40001Z" fill="white"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M3.59998 15.6C3.59998 14.9373 4.13723 14.4 4.79998 14.4H19.2C19.8627 14.4 20.4 14.9373 20.4 15.6C20.4 16.2628 19.8627 16.8 19.2 16.8H4.79998C4.13723 16.8 3.59998 16.2628 3.59998 15.6Z" fill="white"/>
+</g>
+</g>
+</svg>
```

Comment:
You do this with CSS:

```
import React from 'react';
import cx from 'clsx';

import styles from './Hamburger.module.scss';

interface HamburgerProps {
  open: boolean;
  onToggle: () => void;
  classes?: {
    bar?: string;
  };
}

const Hamburger: React.FC<HamburgerProps> = ({ open, onToggle, classes }) => (
  <button type="button" onClick={onToggle} className={cx(styles.container, { [styles.open]: open })}>
    <span className={styles.label}>
      <span className={cx(styles.bar, classes?.bar)} />
      <span className={cx(styles.bar, classes?.bar)} />
      <span className={cx(styles.bar, classes?.bar)} />
    </span>
  </button>
);

export default Hamburger;
```

```

.container {
  height: rem-calc(24);
  border-radius: 0;
  outline: none;
  border: none;
  background-color: transparent;
  transition: all $animation-duration__normal $timing-function;
  cursor: pointer;
  padding: 0;
  margin-left: rem-calc(34);

  @include media(0, 'md') {
    margin-left: rem-calc(24);
  }

  .label {
    display: flex;
    justify-content: space-between;
    flex-direction: column;
    width: rem-calc(24);
    height: rem-calc(16);
    transition: all $animation-duration__normal $timing-function;

    .bar {
      width: 100%;
      height: rem-calc(2);
      background-color: $primary;
      transition: transform $animation-duration__normal $timing-function,
        opacity $animation-duration__normal $timing-function;
    }
  }

  &.open {
    width: rem-calc(34);

    @include media(0, 'md') {
      width: rem-calc(30);
    }

    .label {
      .bar {
        background-color: $secondary;
        &:nth-child(1) {
          transform: rotate(45deg) translate(9px, 0);
        }

        &:nth-child(3) {
          transform: rotate(-45deg) translate(10px, -1px);
        }

        &:nth-child(2) {
          opacity: 0;
        }
      }
    }
  }
}

```

Reply:
:white_check_mark: implemented this with css, used similar files with minor tweaks